### ISSUE TITLE
Support for Colossal size

### ISSUE THREAD

**Issue by Vesselchuck** [Ourobor/Hex-Size-Support#21]
Is there a possibility to add even bigger size for the module? I'm using this exceptional module to host D&D games with hexes, but it lacks Colossal size, sadly.
![hexBasedCreatureSize](https://user-images.githubusercontent.com/70449411/134726518-02f1eb25-27dd-4499-a532-9dba1eb6aad8.png)

**Comment by Ourobor** [comment]
Yeah, sure. Shouldn't be a problem, just a lot of mindnumbing data entry, but it was something I wanted to get to anyway.

My apologies, the module was originally made to support a different system(though it's intentionally agnostic) and I've honestly been surprised how many people end up using it for different stuff. Happy to cater to help out support for other systems though. It's been a long time since I played DND, are there sizes larger than Colossal(Size 5) that would be good to add too?

**Comment by Vesselchuck** [comment]
Yeah, your module is awesome for other systems too, not only for Lancer! Although it's strange that Foundry VTT itself doesn't have support for those things. As far as I know, Colossal is the biggest one.

**Comment by perijove** [comment]
{
		"name": "Size 5",
		"key": 5,
		"height": 5,
		"width": 5,
		"border": [
[-2.5,-0.25],
[-2,-0.5],
[-2,-1],
[-1.5,-1.25],
[-1.5,-1.75],
[-1,-2],
[-0.5,-1.75],
[0,-2],
[0.5,-1.75],
[1,-2],
[1.5,-1.75],
[1.5,-1.25],
[2,-1],
[2,-0.5],
[2.5,-0.25],
[2.5,0.25],
[2,0.5],
[2,1],
[1.5,1.25],
[1.5,1.75],
[1,2],
[0.5,1.75],
[0,2],
[-0.5,1.75],
[-1,2],
[-1.5,1.75],
[-1.5,1.25],
[-2,1],
[-2,0.5],
[-2.5,0.25],
[-2.5,-0.25]
			]
	}

That's the data entry done, if I understand the configuration correctly
